DriveBackup/src/main/java/ratismal/drivebackup/util/FileUtil.java

```diff
@@ -51,14 +51,14 @@ public TreeMap<Long, File> getLocalBackups(String location, LocalDateTimeFormatt
 
                 String fileName = file.getName();
 
-                try {
```

Comment:
Why are we removing this?

Reply:
I think is better if @rhld16 replied this question.